[docs] Describe TestCafe current limitations (#5357)
DevExpress/testcafe: docs/articles/documentation/guides/basic-guides/select-page-elements.md

    Likewise, the return value is the only way to obtain data from selectors.

> * You can not use [property shorthands](http://es6-features.org/#PropertyShorthand) to declare `Selector`'s `dependencies` option.

Reviewer: ```suggestion
* You cannot use [property shorthands](http://es6-features.org/#PropertyShorthand) to declare the `Selector`'s `dependencies` option.
```
